recipes/aqme/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "aqme" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aqme-{{ version }}.tar.gz
+  sha256: 69217bf8059472a2e42603c9f30724d1d9ec22d07c1e3a3fb9e9f4808fd98509
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - ase
+    - cclib
+    - cffi
+    - matplotlib-base
+    - numpy
+    - pandas
+    - progress
+    - python >=3.6
+    - pyyaml
+    - seaborn
+    - openbabel
+    - rdkit
+
+test:
+  imports:
+    - aqme
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jvalegre/aqme
+  summary: Automated Quantum Mechanical Environments
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jvalegre
+    - shreesowndarya

```

Comment:
@shreesowndarya are you ok becoming a maintainer here?

Reply:
Yes.